sample-apps/aws-xray-sdk-go-sample/main.go

```diff
@@ -0,0 +1,67 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+package main
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-xray-sdk-go/xray"
+	"golang.org/x/net/context/ctxhttp"
+	"log"
+	"net/http"
+	"os"
+)
+
+const (
```

Comment:
remove this constant since we are not using?

Reply:
Sure